src/test/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolderTest.java

```diff
@@ -740,6 +840,68 @@ public TopLevelItem newInstance(ItemGroup parent, String name) {
 
     }
 
+    @SuppressWarnings({"unchecked", "rawtypes"})
+    public static class SampleComputedFolderWithWorkflowJobAsChildren extends ComputedFolder<WorkflowJob> {
```

Comment:
Note: not as realistic as using `workflow-multibranch`, but simpler to set up and probably good enough for this purpose.